example/Router.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+use Equip\Queue\Exception\HandlerException;
+use Equip\Queue\Exception\RouterException;
+use Equip\Queue\RouterInterface;
+
+class Router implements RouterInterface
+{
+    /**
+     * @var array
+     */
+    private $routes;
+
+    /**
+     * @param array $routes
+     */
+    public function __construct(array $routes = [])
+    {
+        $this->routes = $routes;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function get($handler)
+    {
+        $route = $this->getRoute($handler);
+        if (!$route) {
+            throw RouterException::routeNotFound($handler);
+        }
+
+        if (is_callable($route)) {
+            return $route;
+        }
+
+        if (class_exists($route)) {
+            return new $route;
+        }
```

Comment:
I'm going with FQCN because I want DI to be able to be used. I'll refactor this as well.